packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -90,7 +90,7 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
         this.onChangeCountryQuery(displayName)
       }
     } catch (error) {
-      console.error(`${TAG}/triggerPhoneNumberRequestAndroid`, 'Could not request phone', error)
+      console.info(`${TAG}/triggerPhoneNumberRequestAndroid`, 'Could not request phone', error)
```

Comment:
This info message seems to be thrown even if able to request the phone number, but the user chooses to input a different number- can you differentiate between these cases, or use a message that would encapsulate both?